pkg/utils/constants.go

```diff
@@ -52,3 +52,18 @@ const ServiceAccountKind = "ServiceAccount"
 
 // OrganizationKind a constant representing the kind of the Organization Model
 const OrganizationKind = "Organization"
+
+// LetsEncryptStaging a constant representing Let's Encrypt staging server url
+const LetsEncryptStaging = "https://acme-staging.api.letsencrypt.org/directory"
```

Comment:
these constants look http-specific, move them to `http` package. In general, avoid `utils` package if possible.